### DISCUSSION TITLE
Is it possible to make default settings for arrays?

### DISCUSSION THREAD

**Discussion by unfamiliartunes** [pierreguillot/Camomile#264] [Q&A]
I have 2 arrays in a patch I'm making (both of which have been programmed for save states). I want one of them to default with all values at 1, and the other with all values at 0. However, I only want that to happen if there aren't any array values already set. How would I go about programming that?

**Comment by pierreguillot** [comment]
I think it should work using [loadbang], did you try? 
<img width="163" alt="Sans titre" src="https://user-images.githubusercontent.com/1409918/119561284-43391500-bda5-11eb-8631-8e9e110431d6.png">

**Reply by unfamiliartunes** [reply]
Perhaps I can just send you the patch? It's far more complicated than that!

**Reply by pierreguillot** [reply]
I'm sorry I can't debug your patch. I think the solution I exposed should work as in Pd.
